### PR TITLE
Add recipe for alv, a command line alignment viewer

### PR DESCRIPTION
**Pull request by mptrsen**
Add recipe for alv, a command line alignment viewer

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by mptrsen** [comment]
@BiocondaBot please add label

File: recipes/alv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - colorama >=0.3.8
+    - python >=3.6
+
+test:
+  commands:
+    - "alv --help"
+
+about:
+  home: "https://github.com/arvestad/alv"
+  license: "GNU General Public v3 (GPLv3)"
```
Comment (bgruening) [inline recipes/alv/meta.yaml:34]: Can you please use those identifiers: https://spdx.org/licenses/GPL-3.0.html

File: recipes/alv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  license_family: GPL3
+  license_file: "LICENSE"
+  summary: "A console-based sequence alignment viewer"
+  doc_url:
```
Comment (bgruening) [inline recipes/alv/meta.yaml:38]: can you clean that up please?

File: recipes/alv/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: 53729576e59e3e99713d75fc43703c3170a115d951f00474f66b29a62062ac8e
+
+build:
+  number: 0
+  noarch: generic
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - biopython >=1.70
```
Comment (bgruening) [inline recipes/alv/meta.yaml:19]: I think only pip and python are needed in the host section

**Review by bgruening — APPROVED** [review]

File: recipes/alv/meta.yaml (hunk 3 above)
Comment (mptrsen) [reply recipes/alv/meta.yaml:19]: I'll have to check this on Monday. Thanks for looking over the yaml!

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**